app/views/choice_page.scala.html

```diff
@@ -14,41 +14,78 @@
-            inputs = Seq(
-                RadioOption("arrival", Arrival.value, messages(s"movement.choice.arrival.label")),
-                RadioOption("associate", AssociateUCR.value, messages(s"movement.choice.associate.label")),
-                RadioOption("disassociate", DisassociateUCR.value, messages(s"movement.choice.disassociateDucr.label")),
-                RadioOption("shut_mucr", ShutMUCR.value, messages("movement.choice.shutMucr.label")),
-                RadioOption("departure", Departure.value, messages(s"movement.choice.departure.label")),
-                RadioOption("submissions", Submissions.value, messages("movement.choice.submissions.label"))
-            )
-        )
+@govukLayout(
+    title = Title(messages("movement.choice.title")),
+    backButtonTitle = messages("site.back"),
+    backButtonUrl = controllers.routes.StartController.displayStartPage().url) {
 
-        @components.submit_button(messages("site.continue"))
+    @formHelper(action = routes.ChoiceController.submitChoice(), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
+        @govukRadios(Radios(
+            name = "choice",
+            fieldset = Some(Fieldset(
+                legend = Some(Legend(
+                    content = Text(messages("movement.choice.title")),
+                    isPageHeading = true,
+                    classes = "govuk-fieldset__legend--xl"
+                ))
+            )),
+            items = Seq(
+                RadioItem(
+                    id = Some(Arrival.toString.toLowerCase),
+                    value = Some(Arrival.value.toString),
+                    content = Text(messages(s"movement.choice.${Arrival.toString.toLowerCase}.label")),
+                    checked = form("choice").value.contains(Arrival.toString.toLowerCase)
+                ),
+                RadioItem(
+                    id = Some("associate"),
+                    value = Some(AssociateUCR.value.toString),
+                    content = Text(messages(s"movement.choice.${AssociateUCR.toString.toLowerCase}.label")),
+                    checked = form("choice").value.contains(AssociateUCR.value.toString)
+                ),
+                RadioItem(
+                    id = Some("disassociate"),
+                    value = Some(DisassociateUCR.value.toString),
+                    content = Text(messages(s"movement.choice.${DisassociateUCR.toString.toLowerCase}.label")),
+                    checked = form("choice").value.contains(DisassociateUCR.value.toString)
+                ),
+                RadioItem(
+                    id = Some("shut_mucr"),
+                    value = Some(ShutMUCR.value.toString),
+                    content = Text(messages(s"movement.choice.${ShutMUCR.toString.toLowerCase}.label")),
+                    checked = form("choice").value.contains(ShutMUCR.value.toString)
+                ),
+                RadioItem(
+                    id = Some(Departure.toString.toLowerCase()),
```

Comment:
Why do you use `Departure.toString` here but above you have hard-coded Strings?

Reply:
I was trying to avoid hardcoding all values, but probably it is better to be consistent